.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 // -*- jsonc -*-
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
accidentally committed?

Reply:
it's from a vscode update that hasn't gone through yet for this repo. studio has this as well.